rust/core/src/project_to_sql.rs

```diff
@@ -1475,6 +1467,7 @@ sources:
             project_root,
             model,
             None,
+            "shift_hours.chart.sql",
```

Comment:
nit: This doesn't quite align with the expected behaviour. 

nit: Might be worth putting in a fake test and validating the error message to make it's something we count on.